tests/gold_tests/next_hop/strategies_ch2/strategies_ch2.test.py

```diff
@@ -73,7 +73,7 @@
     'proxy.config.http.cache.http': 0,
     'proxy.config.http.uncacheable_requests_bypass_parent': 0,
     'proxy.config.http.no_dns_just_forward_to_parent': 1,
-    'proxy.config.http.parent_proxy.mark_down_hostdb': 1,
+    'proxy.config.http.parent_proxy.mark_down_hostdb': 0,
```

Comment:
mark_down_hostdb should always be 0 when using strategies.yaml.  This change is not related to the purpose of this PR, just threw it in to avoid a 1-line PR.